src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionRoutes.scala

```diff
@@ -55,9 +55,11 @@ trait GoogleExtensionRoutes extends ExtensionRoutes with UserInfoDirectives {
               }
             }
           } ~
-          path("proxyGroup") {
+          path("proxyGroup" / Segment) { targetUserEmail =>
```

Comment:
Why are we changing this api? From a security perspective I don't like this because any authenticated user can get the proxy for anyone. I don't know how bad that information leak actually is but it feels wrong.

Reply:
Counterpoint: I want to add you (and your pet) to my non-FC-controlled bucket. I need your proxy group in order to do so. Being able to look it up myself means I don't have to ask you (and makes it way easier for UIs to facilitate this)